articles/users/guides/bulk-user-imports.md

```diff
@@ -17,59 +17,185 @@ useCase:
+    }
+  ]
+}
+```
 
-Before you launch the import users job, you must:
+Depending on the status of the user import job, you'll receive a response similar to one of the following:
 
-* [Configure a database connection](/connections/database) to which the users will be imported and enable it for at least one application.
+**Pending**
 
-## Request bulk import
+```json
+{
+  "status": "pending",
+  "type": "users_import",
+  "created_at": "", // TODO: add date
+  "id": "job_abc123",
+  "connection_id": "CONNECTION_ID",
+  "external_id": "EXTERNAL_ID"
+}
+```
 
-The [Create Job to Import Users endpoint](/api/management/v2#!/Jobs/post_users_imports) requires that your `POST` request encoded as type `multipart/form-data`.
+**Processing**
 
-Create a request that contains the following parameters:
+```json
+{
+  "status": "processing",
+  "type": "users_import",
+  "created_at": "", // TODO: add date
+  "id": "job_abc123",
+  "connection_id": "CONNECTION_ID",
+  "external_id": "EXTERNAL_ID",
+  "percentage_done": 0,
+  "time_left_seconds": 0
+}
+```
 
-| Parameter | Description |
-|-----------|-------------|
-| `users` | [File in JSON format](/users/references/bulk-import-database-schema-examples#file-example) that contains the users to import. |
-| `connection_id` | ID of the connection to which users will be inserted. You can retrieve the ID using the [GET /api/v2/connections](/api/management/v2#!/Connections/get_connections) endpoint. |
-| `upsert` | Boolean value; `false` by default. When set to `false`, pre-existing users that match on email address, user ID, or username will fail. When set to `true`, pre-existing users that match on any of these fields will be updated, but only with upsertable attributes. For a list of user profile fields that can be upserted during import, see [User Profile Attributes](/users/references/user-profile-structure#user-profile-attributes). |
-| `external_id` | Optional user-defined string that can be used to correlate multiple jobs. Returned as part of the job status response. |
-| `send_completion_email` | Boolean value; `true` by default. When set to `true`, sends a completion email to all tenant owners when the import job is finished. If you do *not* want emails sent, you must explicitly set this parameter to `false`. |
+**Completed**
 
-If the request works, you will receive a response similar to the following:
+If a job is completed, the job status response will include totals of successful, failed, inserted, and updated records.
 
 ```json
 {
-    "status":"pending",
-    "type":"users_import",
-    "id":"job_abcdef1234567890",
-    "connection":"abcd",
-    "external_id":"contoso"
+  "status": "completed",
+  "type": "users_import",
+  "created_at": "", // TODO: add date
+  "id": "job_abc123",
+  "connection_id": "CONNECTION_ID",
+  "external_id": "EXTERNAL_ID",
+  "location": "https://${account.namespace}/EXAMPLE"
+  // TODO: example of successful/failed/inserted/updated fields
```

Comment:
☝️ Anyone have an example for these fields?

Reply:
@ian-speers I think he'd have one or two

Reply:
These will appear in the summary field (e.g `"summary":{"failed":1,"updated":0,"inserted":0,"total":1}`).